src/NServiceBus.Core/Transports/Learning/DirectoryBasedTransaction.cs

```diff
@@ -59,11 +71,11 @@ public Task Enlist(string messagePath, string messageContents)
             return AsyncFile.WriteText(txPath, messageContents);
         }
 
-        public void Complete()
+        public Task<bool> Complete()
```

Comment:
I assume this was made to return a task for consistency? As this is internal, does that really provide any benefit?